packages/react/src/hooks/useBreakpointValue.ts

```diff
@@ -2,6 +2,7 @@ import { Breakpoint } from '../primitives/types/responsive';
 import { getValueAtCurrentBreakpoint } from '../primitives/shared/responsive/utils';
 import { useBreakpoint } from '../primitives/shared/responsive/useBreakpoint';
 import { useTheme } from './useTheme';
+import { ThemeStylePropKey } from 'src/primitives/types/theme';
```

Comment:
I think this import will fail in a later build.
```suggestion
import { ThemeStylePropKey } from '../primitives/types/theme';
```